covid/question_covid.py

```diff
@@ -90,28 +146,40 @@ def create_argparser():
     parser = create_argparser()
     args = parser.parse_args()
 
-    TOKENIZER = BertTokenizer.from_pretrained('bert-base-uncased')
-    MODEL = BertForQuestionAnswering.from_pretrained('bert-large-uncased-whole-word-masking-finetuned-squad')
+    if args.pre_trained_model_path:
+        with msg.loading("Loading pre trained model"):
+            with open(args.pre_trained_model_path, "rb") as f:
+                covid_q = pickle.load(f)
+        msg.good(f"Loaded {args.pre_trained_model_path}")
+    else:
+        msg.text("Training new model")
+        covid_q = train()
+        msg.good("Trained")
+        with open("models/covid_q.pkl", "wb") as f:
+            pickle.dump(covid_q, f)
 
-    articles_dir = 'data/raw/CORD-19-research-challenge/'
-    articles_folders = [
-        'biorxiv_medrxiv/biorxiv_medrxiv/pdf_json/',
-        'comm_use_subset/comm_use_subset/pdf_json/',
-        'comm_use_subset/comm_use_subset/pmc_json/',
-        'noncomm_use_subset/noncomm_use_subset/pdf_json/',
-        'noncomm_use_subset/noncomm_use_subset/pmc_json/',
-        'custom_license/custom_license/pdf_json/',
-        'custom_license/custom_license/pmc_json/']
-
-    print("Ingesting publications")
-    data_text, index2paperID = get_data_texts(articles_dir, articles_folders)
-
-    print("Fitting TFIDF index")
-    covid_q = QuestionCovid(TOKENIZER, MODEL)
-    covid_q.fit(data_text)
 
     print("Finding best answer")
-    _, best_answer = covid_q.predict(args.question, data_text, index2paperID)
-
-    print("----- Answer: -----")
-    print(best_answer)
+    start = time.time()
+    with open(args.answers_path, "w") as f:
```

Comment:
might want `from datetime import datetime`
and here:
`'data/answers/' + datetime.now().strftime('%Y-%m-%d') + args.answers_path`